cranelift/codegen/meta/src/isa/x86/instructions.rs

```diff
@@ -376,6 +376,40 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    let x = &Operand::new("x", TxN);
+    let y = &Operand::new("y", TxN);
+    let a = &Operand::new("a", TxN);
+
+    ig.push(
+        Inst::new(
+            "x86_punpckh",
+            r#"
+        Unpack the high-order lanes of ``x`` and ``y`` and interleave into ``a``. With notional
+        i8x4 vectors, where ``x = [x1, x2, x3, x4]`` and ``y = [y1, y2, y3, y4]``, this operation
+        would result in ``a = [x3, y3, x4, y4]`` (note that this uses left-to-right lane ordering
```

Comment:
What does `this` refer to here?  On the whole I'd prefer to stick with the Intel style convention, where the rightmost lane is lane zero.  But note; my comment is very "drive-by"; I don't know what the lane numbering conventions in CL are.

Reply:
Ok, we can switch to Intel-style. I went back and forth on this one and I can't recall why I chose this ordering. I'll look around the file to see if there was something that made me choose this.